### PR TITLE
Supprt Aqara HomeKit cube with other Xiaomi Cube(s)

### PR DESCRIPTION
**Pull request by RosemaryOrchard**
`lumi.remote.cagl02` seems identical to `lumi.remote.cagl01`, aside from the dice like markings on the outside.

### CODE REVIEW

File: devices/xiaomi.js [hunk 1]
```diff
@@ -1481,7 +1481,7 @@ module.exports = [
         ota: ota.zigbeeOTA,
     },
     {
-        zigbeeModel: ['lumi.sensor_cube', 'lumi.sensor_cube.aqgl01'],
+        zigbeeModel: ['lumi.sensor_cube', 'lumi.sensor_cube.aqgl01', 'lumi.remote.cagl02'],
```
Comment (Koenkk) [inline devices/xiaomi.js:1484]: It seems this device has a different model: `MFCZQ12LM`, can you duplicate it this definition and also update the description?

**Comment by CrazyCoder** [comment]
I tried to create a new external converter for `lumi.remote.cagl02`, but it didn't seem to work. Only tap action is delivered, sometimes I see this in the log

```
No converter available for 'MFCZQ12LM' with cluster 'aqaraOpple' and type 'attributeReport' and data '{"329":5}'
```

Side of the cube report weird numbers like 512.

Other exposed parameters are null.

Code of the external converter (I hoped that MFKZQ01LM would work, but probably they have to be written from scratch):

```
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');
const e = exposes.presets;
const ea = exposes.access;

const definition = {
    zigbeeModel: ['lumi.remote.cagl02'],
    model: 'MFCZQ12LM',
    vendor: 'Xiaomi',
    description: 'Mi/Aqara Cube T1 Pro',
    meta: {battery: {voltageToPercentage: '3V_2850_3000'}},
    fromZigbee: [fz.xiaomi_basic, fz.MFKZQ01LM_action_multistate, fz.MFKZQ01LM_action_analog],
    exposes: [e.battery(), e.battery_voltage(), e.angle('action_angle'), e.device_temperature(), e.power_outage_count(false),
        e.cube_side('action_from_side'), e.cube_side('action_side'), e.cube_side('action_to_side'), e.cube_side('side'),
        e.action(['shake', 'wakeup', 'fall', 'tap', 'slide', 'flip180', 'flip90', 'rotate_left', 'rotate_right'])],
    toZigbee: [],
};

module.exports = definition;
```

**Comment by andrewgraham1** [comment]
Just got one of these and tested locally.

Can confirm that the MFCZQ12LM is the exact same device, the only difference is the model change and the physical appearance of the device

**Comment by CrazyCoder** [comment]
There is work already underway for proper support at https://github.com/Koenkk/zigbee2mqtt/issues/15652.

**Comment by Gehude1** [comment]
Hi I have this issue can any one help ??

JJPro
/
CTP-R01-converter


[16:36:30] INFO: Preparing to start...
[16:36:31] INFO: Socat enabled
[16:36:31] INFO: Starting socat
Socat startup parameters:
Options:     -d -d
Master:      pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
Slave:       tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
[16:36:31] INFO: Starting socat process ...
2023/02/02 16:36:31 socat[54] N PTY is /dev/pts/0
2023/02/02 16:36:31 socat[54] N listening on AF=2 0.0.0.0:8485
[16:36:34] INFO: Starting Zigbee2MQTT...
/app/data/extension/externally-loaded.js:8
<!DOCTYPE html>
^
SyntaxError: Unexpected token '<'
    at new Script (node:vm:100:7)
    at createScript (node:vm:265:10)
    at Object.runInNewContext (node:vm:306:10)
    at loadModuleFromText (/app/lib/util/utils.ts:150:8)
    at loadModuleFromFile (/app/lib/util/utils.ts:157:12)
    at Object.getExternalConvertersDefinitions (/app/lib/util/utils.ts:167:25)
    at getExternalConvertersDefinitions.next (<anonymous>)
    at new ExternalConverters (/app/lib/extension/externalConverters.ts:12:20)
    at new Controller (/app/lib/controller.ts:84:58)
    at start (/app/index.js:106:18)

**Pull request closed without merging**